changelog/issue-3652.md

```diff
@@ -0,0 +1,10 @@
+audience: users
+level: major
+reference: issue 3652
+---
+
+It is now possible to cancel sealed task group by using `cancelTaskGroup` method.
+This will cancel all scheduled/pending/running tasks within given group.
+
+`taskcluster group cancel` was removed from shell client,
+as it is available under `taskcluster api queue cancelTaskGroup` now.
```

Comment:
Good point. 
Maybe worth then keeping the old version because it was doing something different. But probably add a warning that this method is deprecated to give anyone who's using it heads up. 👍